wolox-books/src/app/screens/auth/auth.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="navbar">
+  <div class="container">
+    <img src="../../../../../assets/wolox_books_logo.png" alt="WOLOX Books">
+    <button type="button" class="logout" (click)="this.user.logout()">Logout</button>
```

Comment:
Dont access the user service directly from the view. Create a method in the component and make that method make the call. That way youll be able to subscribe to its response and resirect the user to the login